.github/workflows/assets.yml

```diff
@@ -1,26 +1,36 @@
-name: Compile assets
+name: Compile dist
 
 on:
-  push:
-    paths:
-      - 'www/assets/**'
-  pull_request:
+  pull_request_review:
+    types:
+      - submitted
     paths:
       - 'www/assets/**'
```

Comment:
If I read the docs correctly, the `pull_request_review` event does not support the `paths` option, so the workflow will run on all approved PRs. I don't know if there is a way around that.

Reply:
I thought about this again and think it's fine that the workflow always runs on all PRs: If the build script only changes the dist files if the src files changed (i.e. if the dist files stay the same if the src is unchanged), it's a non-issue to run the workflow regardless as it won't result in a commit.

Reply:
You are right, it's not really an issue. Just annoys me that it will run even for PRs the only change a content file XD

Reply:
I'm curious: Why does it annoy you?

Reply:
Cause it's running a check that's totally unnecessary for the PR then. And it delays showing the status of the Pr as the test hast to be finished first. It's a non-issue as you say, but it bothers me especially because it would be so easy to define the rule for when it should run and when not. But I'll be able to live with it ;)